library-best-practices/JLBP-6.md

```diff
@@ -17,9 +17,7 @@ access classes in library A:
    generally share a Java package (the package as defined in `package`
    statements, for example `package com.google.common.collect`). The classpath,
    which is formed from Maven artifacts and possibly non-Maven sources, is
-   searched for each fully-qualified class name at runtime. In order to simplify
-   the discussion, we only talk about the Java package, assuming that the
```

Comment:
I think this clarification was added because one of the readers of the JLBP was confused, and this needed to be clarified.

Reply:
Except that this sentence is actively incorrect. It says "we only talk about the Java package..." and then immediately proceed to talk about the Maven coordinates too.

If this is trying to say something else, it's unclear and I did not follow it so it should be rephrased.

Reply:
Oh, we were trying to say that as far as the fully-qualified class name goes, we only talk about the Java package part of that fully-qualified class name. (The scope of the comment was limited to that.)

Reply:
I don't see that we use that distinction below.